packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -1416,6 +1420,9 @@ class _NestedScrollPosition extends ScrollPosition implements ScrollActivityDele
     if (simulation == null) {
       return IdleScrollActivity(this);
     }
+    final bool shouldIgnorePointer = !outOfRange
```

Comment:
I feel like I may have fatigued on this, why would we want to ignore the pointer if we are not out of range?

Reply:
To allow "tap to stop" the scrollable, otherwise it would stop the scrollable and also register a tap on the inner child, e.g. a button. (If that is what you mean)

Reply:
Ah ok, thank you, sorry I got mixed up on that. Should `shouldIgnorePointer` this be a getter in the ScrollPosition super class? That way we don't have to repeat `!outOfRange && (activity?.shouldIgnorePointer ?? true)` here and in scroll_position.dart

Reply:
This `shouldIgnorePointer` getter and the nits in the tests files should be it here and we can land this! Thank you for your work on this. We are extra careful with things like scrolling since it can regress in pretty wild and crazy ways. :)